pygyw/layout/drawings.py

```diff
@@ -82,10 +82,10 @@ class TextDrawing(GYWDrawing):
 
     def __init__(self,
                  text: str,
+                 font: fonts.GYWFont = fonts.GYWFonts.ROBOTO_MONO,
+                 size: int = 24,
```

Comment:
Why did you reorder these two ?

Reply:
Idk, the order seemed more logical. I can change it back if you don't like it.

Reply:
I prefer to have `left` and `top` as first optional parameter for every drawing, yes